raiden_contracts/tests/test_deploy_script.py

```diff
@@ -85,6 +85,12 @@ def deployed_raiden_info(deployer: ContractDeployer) -> DeployedContracts:
     return deployer.deploy_raiden_contracts(max_num_of_token_networks=1)
 
 
+@pytest.mark.slow
+@pytest.fixture(scope="session")
+def deployed_raiden_info2(deployer: ContractDeployer) -> DeployedContracts:
```

Comment:
Not a very informative name ;)

Reply:
True.  Though I just needed another one.